dist/less/ds4/mixins.less

```diff
@@ -603,14 +648,29 @@
     .background-icon-image(@width; @height; @ds4-icon-settings-base64);
 }
 
+.icon-filter-gallery(@width: @ds4-icon-filter-gallery-width; @height: @ds4-icon-filter-gallery-height) {
+    .background-icon-image(@width; @height; @ds4-icon-filter-gallery-base64);
+}
+
+.icon-filter-list(@width: @ds4-icon-filter-list-width; @height: @ds4-icon-filter-list-height) {
+    .background-icon-image(@width; @height; @ds4-icon-filter-list-base64);
+}
+
+.icon-filter-single(@width: @ds4-icon-filter-single-width; @height: @ds4-icon-filter-single-height) {
+    .background-icon-image(@width; @height; @ds4-icon-filter-single-base64);
+}
+
+// deprecated in favour of filter-grid
```

Comment:
Should be `filter-gallery`

Reply:
I didn't see this fixed, but if you're going to remove these deprecated ones then it won't even matter.

Reply:
I changed it in the src. Maybe I forgot to build the dist file...